config/routes.php

```diff
@@ -1,4 +1,4 @@
 <?php
-use Cake\Routing\Router;
-
-Router::extensions(['pdf']);
```

Comment:
@ndm2 Since this file is basically a no-op when plugin routes are added at the let's just keep it this way. On the off chance that someone has moved the statement to load plugin routes at the top it won't break that application.